code/modules/admin/admin_verbs.dm

```diff
@@ -665,6 +665,16 @@ var/list/admin_verbs_mod = list(
 			var/heavy_impact_range = input("Heavy impact range (in tiles):") as num
 			var/light_impact_range = input("Light impact range (in tiles):") as num
 			var/flash_range = input("Flash range (in tiles):") as num
+			if(devastation_range > 299 || heavy_impact_range > 299 || light_impact_range > 299)
```

Comment:
if(max(devastation_range,heavy_impact_range,light_impact_range)>299) is slightly more elegant

you don't need to update this pr, just advice for the future since it makes it much easier to change the threshold in one go